modules/http-api/src/main/java/org/mule/runtime/http/api/domain/message/HttpMessageBuilder.java

```diff
@@ -40,6 +40,17 @@ public B setEntity(HttpEntity entity) {
   }
 
   /**
+   * @param headersMap a {@link MultiMap} representing the HTTP headers of the {@link HttpRequest} desired. Non null.
+   * @return this builder
+   */
+  public B setHeaders(MultiMap<String, String> headersMap) {
```

Comment:
shouldn't we rename this to `headers` and also make the other setters consistent?